src/main/java/com/faforever/moderatorclient/ui/main_window/LadderMapPoolController.java

```diff
@@ -40,19 +44,18 @@
 import javafx.scene.image.ImageView;
 import javafx.scene.layout.HBox;
 import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
 import javafx.util.StringConverter;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.compress.utils.Lists;
 import org.apache.maven.artifact.versioning.ComparableVersion;
 import org.springframework.core.io.ClassPathResource;
 import org.springframework.stereotype.Component;
 
-import java.io.IOException;
+import java.io.*;
```

Comment:
Personally I prefer not to use star imports to reduce name collisions

Reply:
Done.

Reply:
There are still some on import com.faforever.commons.api.dto.*;
This is a setting you can do in your IDE. Many IDEs like to summarize them to wildcard imports :(